recipe/meta.yaml

```diff
@@ -40,20 +26,21 @@ requirements:
     - make
     - pkg-config
     - {{ compiler('cxx') }}
+    - llvmdev {{ llvm_version }}.*       # [build_platform != target_platform]
+    - clangdev {{ llvm_version }}.*      # [build_platform != target_platform]
   host:
     - ocl-icd                            # [linux]
     - khronos-opencl-icd-loader          # [osx]
     - libhwloc
     - llvmdev {{ llvm_version }}.*
     - clangdev {{ llvm_version }}.*
     - llvm {{ llvm_version }}.*
-    - lld {{ llvm_version }}.*
     - libclang-cpp {{ llvm_version }}.*
     - zlib                               # [osx]
+    - ld64 >=530                         # [osx]
   run:
     - ocl-icd                            # [linux]
     - khronos-opencl-icd-loader          # [osx]
-    # TODO: Remove this when LLVM 10.0.1 is released
     - ld64 >=530                         # [osx]
```

Comment:
@isuruf Should this be removed (and not just the comment?)

Reply:
No, it should remain due to issues on Big Sur.

Reply:
K, thx. Just checking.